### PR TITLE
python3Packages.i3pystatus: drop praw

### PR DESCRIPTION
**Pull request by teto**
praw is a reddit wrapper that sometimes poses problems when installing
i3pystatus (twice in the past 6 months for me).
As its usage should be relatively low, it should be fine to drop.
users can go back the old behavior by overriding extraLibs

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by teto** [comment]
follow up of https://github.com/NixOS/nixpkgs/issues/64086 CC @igsha

File: pkgs/applications/window-managers/i3/pystatus.nix [hunk 1]
```diff
@@ -3,19 +3,19 @@
 python3Packages.buildPythonApplication rec {
   # i3pystatus moved to rolling release:
   # https://github.com/enkore/i3pystatus/issues/584
-  version = "unstable-2019-02-10";
+  version = "unstable-2019-06-10";
   pname = "i3pystatus";
   disabled = !python3Packages.isPy3k;
```
Comment (jonringer) [inline pkgs/applications/window-managers/i3/pystatus.nix]: this will always resolve to `disabled = false`, i would just remove it.
```suggestion
```

**Review by jonringer — COMMENTED** [review]
`$ nix-review pr 64103` passes
otherwise, diff LGTM

```
[3 built, 4 copied (0.7 MiB), 0.2 MiB DL]
https://github.com/NixOS/nixpkgs/pull/64103
1 package were build:
i3pystatus
```

**Comment by teto** [comment]
thanks for the review/suggestion !

**Pull request merged**